.github/workflows/ci.yml

```diff
@@ -76,7 +76,7 @@ jobs:
 
       - name: notify slack
         uses: 8398a7/action-slack@v2
-        if: always() # Pick up events even if the job fails or is canceled.
+        if: github.ref == 'refs/heads/canary'
```

Comment:
The problem with this is it will only pick up the event when it succeeds, which defeats the purpose.

I'll still merge, but I'll also update the config on the `canary` branch.